.docker.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -x
+
+env
+storage-ng-travis-cpp && bash <(curl -s https://codecov.io/bash)
```

Comment:
So apparently we need some extra information on the header of `.docker.sh` explaining what is it used for.